test/quantization/core/test_quantized_op.py

```diff
@@ -4223,6 +4223,82 @@ def test_qlinear_pt2e(self):
                     y_s: {y_scale}, y_zp: {y_zp}""",
                 )
 
+    @skipIfNoONEDNN
+    def test_qlinear_pt2e_gelu(self):
```

Comment:
nit: test_qlinear_gelu_pt2e

Reply:
Can we merge the common parts of this UT and `test_qlinear_pt2e`? Looks like lots of duplicated code.